LoRaEngine/modules/LoRaWanNetworkSrvModule/LoRaWan.NetworkServer/BasicsStation/DownstreamSender.cs

```diff
@@ -86,7 +86,7 @@ private string Message(DownlinkPktFwdMessage message, Region region)
                 writer.WriteNumber("RX1DR", dataRate);
                 writer.WriteNumber("RX1Freq", (ulong)(message.Txpk.Freq * 1e6));
                 writer.WriteNumber("RX2DR", region.GetDefaultRX2ReceiveWindow().DataRate);
-                writer.WriteNumber("RX2Freq", (ulong)(region.GetDefaultRX2ReceiveWindow().Frequency * 1e6));
+                writer.WriteNumber("RX2Freq", (ulong)region.GetDefaultRX2ReceiveWindow().Frequency);
```

Comment:
Instead of casting to `ulong`, could we use `AsUInt64` here?

Reply:
What do you see as the difference/benefit of one over the other. To me, the cast is as explicit as having to use a property.

Reply:
Makes sense, thanks for clarifying